exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -35,7 +35,8 @@ func createMetricsExporter(ctx context.Context, set exporter.CreateSettings,
 		return nil, errors.New("invalid configuration")
 	}
 
-	prwe, err := newPRWExporter(prwCfg, set)
+	telemetry := newPRWTelemetry(set)
+	prwe, err := newPRWExporter(prwCfg, set, telemetry)
```

Comment:
Should this be done inside `newPRWExporter()`?  It's already receiving the settings object `newPRWTelemetry()` needs and the telemetry object isn't used outside of the new exporter function.